README.md

```diff
@@ -136,7 +136,7 @@ In order to build cloud native, security minded and dependable services, there e
 ### Versions
 | vagrant-hashistack | terraform | consul | nomad  | vault  | packer  | consul-template | remarks                  |
 | ------------------ | :-------- |:-------|:-------|:-------| :------ |:----------------|:-------------------------|
-| 0.8.0              |  0.13.5   | 1.8.5  | 0.12.8 | 1.6.0  | 1.6.5   | 0.25.1          |                          |
+| 0.9.0              |  0.13.5   | 1.8.5  | 0.12.8 | 1.6.0  | 1.6.5   | 0.25.1          |                          |
```

Comment:
Did we skip version `0.8.0`?

Reply:
Yes we did. https://app.vagrantup.com/fredrikhgrelland/boxes/hashistack
#456